code/modules/mob/living/carbon/human/species/station/vaurca/vaurca_subspecies.dm

```diff
@@ -170,4 +170,72 @@
 
 /datum/species/bug/type_big/handle_post_spawn(var/mob/living/carbon/human/H)
 	H.mutations.Add(HULK)
-	return ..()
\ No newline at end of file
+	return ..()
+
+/datum/species/bug/type_e
+	name = SPECIES_VAURCA_BULWARK
+	short_name = "vak"
+	name_plural = "Type E"
+	bodytype = BODYTYPE_VAURCA_BULWARK
+	preview_icon = 'icons/mob/human_races/vaurca/r_vaurcae.dmi'
+	icon_template = 'icons/mob/human_races/vaurca/r_vaurcae.dmi'
+	icobase = 'icons/mob/human_races/vaurca/r_vaurcae.dmi'
+	deform = 'icons/mob/human_races/vaurca/r_vaurcae.dmi'
+
+	icon_x_offset = -9
+	healths_x = 22
+	healths_overlay_x = 9
+
+	damage_overlays = 'icons/mob/human_races/masks/dam_mask_bulwark.dmi'
+	damage_mask = 'icons/mob/human_races/masks/dam_mask_bulwark.dmi'
+	blood_mask = 'icons/mob/human_races/masks/dam_mask_bulwark.dmi'
+	onfire_overlay = 'icons/mob/OnFire_bulwark.dmi'
+
+	eyes_icons = 'icons/mob/human_face/eyes48x48.dmi'
+	eyes = "bulwark_eyes"
+
+	slowdown = 6
+
+	unarmed_types = list(/datum/unarmed_attack/claws/vaurca_bulwark)
+
+	natural_armor = list(
+		melee = ARMOR_MELEE_SMALL
+	)
+
+	brute_mod = 0.65
+	burn_mod = 1
+	oxy_mod = 1
+	radiation_mod = 0
+	toxins_mod = 3
+
+	grab_mod = 0.8
+	resist_mod = 4
+
+	spawn_flags = IS_RESTRICTED
+
+	mob_size = 12
+	taste_sensitivity = TASTE_DULL
+	blurb = {"Type E Vaurca, otherwise known as the Bulwarks, are a new bodyform derived from the worker caste in a collaboration by the C’thur and Jargon scientists. Originally only the C’thur had access to these behemoths, but after a short amount of time, the bodyform started appearing in the ranks of the Zo’ra and K’lax as well, causing an even more strained relationship between the hives.<br>
+Similar to Workers, Bulwarks are generally passive, and prefer to flee a fight rather than resist. Though due to their speed, they may still choose to defend themselves should they be unable to properly escape a battle. The main exception to this is when another Vaurca is in danger. When this occurs, they tend to put themselves in between the attacker and the Vaurca, acting as a shield of sorts. They won’t go out of their way to take down the attacker, but will ensure the others get away safely.<br>
+Bulwarks are much larger and have significantly thicker carapaces than most Vaurca, making them slow but resistant to most hits, including a complete immunity to radiation. Their powerful arms and claws grant them stronger punches, enough to bend metal, and is often used to pry open non-functional doors.<br>
+<b>Type E Vaurca are typically used for heavy lifting, agricultural and industrial work, thus they can typically be found as Engineers, Cargo Technicians, Miners and similar jobs, though they can also be found in positions such as janitor and Assistant should they be needed.</b>"}
+
+	heat_level_1 = 360 //Default 360
+	heat_level_2 = 400 //Default 400
+	heat_level_3 = 800 //Default 1000
+
+	sprint_speed_factor = 1.4
```

Comment:
How fast does this make them go, roughly? What's their stamina like?

Reply:
This is G2 speed, but they have vaurca-level stamina